tests/test_price_prediction.py

```
@@ -1,595 +1,149 @@
+            "timeframes": ["5m"],
+            "update_interval_seconds": 1,
+            "cache_ttl_seconds": 30,
+        },
+    )
+    return engine
+
+
+@pytest.mark.asyncio
+async def test_price_engine_updates_prediction_cache(price_engine: AdvancedPricePredictionEngine) -> None:
+    symbol = "BTC/USDT:USDT"
+    await price_engine._update_predictions([symbol])  # noqa: SLF001 - exercising internal helper
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `SLF001`)

Remove unused `noqa` directive

(RUF100)